exporter/elasticsearchexporter/elasticsearch_bulk.go

```diff
@@ -128,6 +133,20 @@ func newElasticsearchClient(logger *zap.Logger, config *Config) (*esClientCurren
 	})
 }
 
+func newRetryOnError(_ *http.Request, err error) bool {
```

Comment:
I think if `config.Retry.Enabled` we should always return false here, since the user has explicitly disabled retries. What do you think?

Reply:
good point. I will take that adjustment in action.

Reply:
Done :)